### PR TITLE
Improved content formatting

### PR DESCRIPTION
**Pull request by kshitij10496**
Added assignments and improved content structuring

### CODE REVIEW

**Comment by kshitij10496** [comment]
@OrkoHunter  Please review and merge

File: index.html [hunk 1]
```diff
@@ -1,26 +1,33 @@
 <html>
   <head>
-    <title>Welcome to the homepage of the batch of MA '19</title>
-  </head>
+    <title>Homepage of MA-2k19</title>
+  </head>'/home/kshitij/Repo/kgpmnc.github.io'
```
Comment (OrkoHunter) [inline index.html]: Why do we need this change?

**Comment by OrkoHunter** [comment]
Line no. 22-30

``` diff
+<<<<<<< HEAD
+        <li> <b>Monday, September 28, 2015</b> : <a href="https://github.com/kgpmnc/kgpmnc.github.io/blob/master/Ass/DAA/Assignment_3.pdf?raw=true">DAA Assignment 3 </a> <a href="https://github.com/kgpmnc/kgpmnc.github.io/blob/master/Ass-Sol/DAA/ass3sol.pdf?raw=true">Solutions</a></li> 
+        <li> <b>Monday, September 28, 2015</b> : <a href="https://github.com/kgpmnc/kgpmnc.github.io/blob/master/Ass/DAA/Assignment_4.pdf?raw=true">DAA Assignment 4</a><a href="https://github.com/kgpmnc/kgpmnc.github.io/blob/master/Ass-Sol/DAA/ass4sol.pdf?raw=true"> Solutions</a></li>
+        <li> <b>Monday, October 12, 2015</b> : <a href="https://github.com/kgpmnc/kgpmnc.github.io/blob/master/Ass/TC/Assignment_2.pdf?raw=true">Transform Calculus Assignment 2 </a></li>
+=======
         <li> <b>Monday, September 28, 2015</b> : DAA Assignment 3<a href="https://github.com/kgpmnc/kgpmnc.github.io/blob/master/notes/DAA/ass3sol.pdf?raw=true"> Solutions</a></li> 
         <li> <b>Monday, September 28, 2015</b> : DAA Assignment 4<a href="https://github.com/kgpmnc/kgpmnc.github.io/blob/master/notes/DAA/ass4sol.pdf?raw=true"> Solutions</a></li>
         <li> <b>Monday, October 12, 2015</b> : Transform Calculus Assignment 2 </li>
+>>>>>>> upstream/master
```

This is the result of merge conflict. And for _solving_ it, you need to pick on of the two parts separate by `=======` and delete everything else in these lines. This shows that the head is confused which one of the revisions to choose and hence is showing both.

Make the changes, amend it and then `push -f`

**Comment by kshitij10496** [comment]
@OrkoHunter What about now ?

File: index.html (hunk 1 above)
Comment (kshitij10496) [inline index.html]: I think this is due to the merge conflict.

**Comment by OrkoHunter** [comment]
Looks good to me.

**Comment by kshitij10496** [comment]
Cool

**Pull request merged**